packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2485,6 +2485,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
             let handle: string;
             try {
+                this.logger.sendTelemetryEvent({ eventName: "uploadSummaryWithContext",
```

Comment:
I believe we had this telemetry here and removed it because it was a duplicate of the one in the driver. I would suggest adding this in the driver for this reason.
@jatgarg @vladsud probably have more details on this.

Reply:
We may already have something like this in the routerlicious driver. @jatgarg ?

Reply:
I think we only log, for ODSP,  if the proposal handle is not what we expected. We could add something similar to the FRS, thoughts ?  
 if (this.lastSummaryProposalHandle !== undefined &&
            this.lastSummaryProposalHandle !== context.proposalHandle) {
            this.mc.logger.sendTelemetryEvent({
                eventName: "LastSummaryProposedHandleMismatch",
                ackedSummaryProposedHandle: context.proposalHandle,
                lastSummaryProposalHandle: this.lastSummaryProposalHandle,
            });
        }

Reply:
We do have this in routerlicious driver. Not sure if it applies to AFR - https://github.com/microsoft/FluidFramework/blob/ac07ab603b2994ddeccad37e835577bbc332bddc/packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts#L104

This is what we have for ODSP - https://github.com/microsoft/FluidFramework/blob/ac07ab603b2994ddeccad37e835577bbc332bddc/packages/drivers/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts#L154

Reply:
We could add these in driver in same events by extracting all these from the summaryContext. But then we may need to edit that in all drivers. Maybe since you are debugging, we could add this in runtime temporarily and then remove later on once we don't need it? You can have a comment or task opened up to remove this event later on?